src/ansys/edb/core/inner/messages.py

```diff
@@ -934,9 +934,7 @@ def net_ref_message(net):
 
 def adaptive_frequency_message(frequency: str, max_delta_s: float, max_passes: int):
```

Comment:
@drewm102 Seems like we don't need this parameter anymore

Reply:
@chenchienjacklin It looks like this method isn't used anywhere so I'll just go ahead and remove it